_layouts/dataset.html

```diff
@@ -49,7 +52,7 @@
       <dd>
         {% if page.category.size %}
           {% for category in page.category %}
-            <a href="{{ site.baseurl }}/datasets/?category={{ category | slugify }}">{{ category }}</a><br>
+            <a href="{{ site.baseurl }}/datasets/?category={{ category | slugify }}"><h4><span class="label label-danger">{{ category }}</span></a></h4><br>
```

Comment:
Somehow this seems to be putting large gaps between the categories when there are multiple. Perhaps it's the `<h4>` tag?
<img width="231" alt="screen shot 2016-04-01 at 10 01 05" src="https://cloud.githubusercontent.com/assets/761444/14209173/d406184c-f7f0-11e5-9b5d-6e26b944bb72.png">